content/ecs_advance_schedule/clean_up.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Clean Up
+chapter: false
+weight: 60
+---
+
+Delete the CloudFormation stack created for this workshop.
+
+```
+$ aws cloudformation delete-stack --stack-name ecs-demo
```

Comment:
There are some CFN issues when deleting capacity providers with active tasks (ie the asg will never delete the instances because of termination protection). I think you may need to add a cli command to turn off managed instance protection on the ASG.